docs/viplab3.0/computation_template.md

```diff
@@ -76,23 +77,31 @@ Note: `//` with text following until EOL is a comment,
   "parameters" : // parameters can be used to supply values at runtime to the configuration
   [
     {
-      "mode" : "any", // depending on guiType either any or fixed
-      "identifier" : "__STEPWIDTH__", 
-      "metadata" : {
-        "guiType" : "input_field",
-        "type": "number",
+      "mode": "fixed", // in this section, the parameters can only be of mode fixed
+      "identifier": "__STEPWIDTH__",
+      "metadata": {
+        "guiType": "dropdown",
         "name": "stepwidth",
-        "decription" : "This section describes the parameter",
+        "decription" : "Stepwidth that is added, when function is called",
```

Comment:
Verstehst du mit dieser Description was stepwidth tut? Für mich ist das immer noch kryptisch.
Vorschlag: die Aufgabe ist ja eine Funktion zu schreiben, die "bar" ausgibt.
Dann könnte ein Parameter ja z.B. sein, in welcher Schriftgröße? (wobei ich nihct weiß ob man bei stdout die schriftgröße beeinflussen kann... dann vielleicht ob es eine Einrückung gibt? oder die Buchstaben mit Spaces auseinandergezogen werden? oder oder oder
-> es geht darum ein wirklich sinnvolles Beispiel zu erstellen